BigData/BigDataTutorial.md

```diff
@@ -9,12 +9,15 @@
 - [Official Apache Hadoop Site](https://hadoop.apache.org) - Apache Hadoop Site
 - [edureka Courses](https://www.youtube.com/user/edurekaIN/playlists?sort=dd&shelf_id=6&view=50) - Edureka's All Youtube Videos
 - [Online Competition For Data Science](https://www.kaggle.com/) - Kaggle
-- [Mapr Tutotial](http://learn.mapr.com/) - One of the Best Tutorial For Big Data and Hadoop
+- [MapR Tutotial](http://learn.mapr.com/) - One of the Best Tutorial For Big Data and Hadoop
 - [Big Data Projects](https://www.acquia.com/examples-big-data-projects) - Big Data Project Ideas
 - [HDFS Architechture](https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html) - Hadoop Distributed File System Arch.
-- [Mapr  Papers](https://www.mapr.com/resources/white-papers) - Hadoop White Papers
+- [MapR  Papers](https://www.mapr.com/resources/white-papers) - Hadoop White Papers
```

Comment:
I didn't add these links. However, "Big Data" folder doesn't have folder structure. That is why I included all the links in one.